packages/nimble-components/src/mapping/icon/tests/mapping-icon.stories.ts

```diff
@@ -0,0 +1,43 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../../utilities/tests/storybook';
+import { hiddenWrapper } from '../../../utilities/tests/hidden';
+import { mappingKeyDescription } from '../../base/tests/story-helpers';
+
+const metadata: Meta = {
+    title: 'Tests/Mappings',
+    parameters: {
+        docs: {
+            description: {
+                component:
+                    'The `nimble-mapping-icon` element defines a mapping from a data value to an icon representation to use for that value. It is meant to be used as content of the `nimble-table-column-icon` element.'
+            }
+        }
+    }
+};
+
+export default metadata;
+
+export const iconMapping: StoryObj = {
+    render: createUserSelectedThemeStory(hiddenWrapper(html`<style></style>`)),
+    argTypes: {
+        key: {
+            description: mappingKeyDescription('the mapped icon'),
+            control: { type: 'none' }
+        },
+        icon: {
+            control: { type: 'none' },
+            description:
+                'The tag name of the Nimble icon to render, e.g. `nimble-icon-check`.'
+        },
+        severity: {
+            description:
+                'One of `error`, `warning`, `success`, `information`, or unspecified. Controls the color of the icon.'
```

Comment:
Sure, I guess it's better for them to use that type instead of string literals. I just referenced it by name. I don't think there's anything helpful to actually link to.